WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabViewController.swift

```diff
@@ -10,6 +11,8 @@ class ReaderTabViewController: UIViewController {
         return makeReaderTabView(viewModel)
     }()
 
+    private var searchButton: SpotlightableButton?
```

Comment:
Optional suggestion: the search button isn't really optional. I'd consider making this var non-optional (either by making it lazily-initialized, or just creating it as `SpotlightableButton(type: .custom)`).

It's not a big deal, but it makes the code clearer IMHO.